Browser_IDE/IDBStoredProject.js

```diff
@@ -78,13 +108,13 @@ class __IDBStoredProjectRW{
         let IDBFS = this;
         return new Promise(function(resolve, reject){
 
-            if (IDBFS.owner.projectName == null)
+            if (IDBFS.owner.projectID == null)
                 return reject();
 
             if (IDBFS.db != null)
                 reject();
 
-            let openRequest = indexedDB.open(IDBFS.owner.projectName, 1);
+            let openRequest = indexedDB.open("SplashKitOnlineProject_" + IDBFS.owner.projectID, 1);
```

Comment:
I agree on using a constant. I'm not sure what you mean w/r/t incorporating the prefix into the ID, though; I don't think it would help handle old projects. The problem there is that old projects only exist as free-floating databases, and are not listed in the new `userProjects` object store. The only way we could know they exist is by going through all the databases in storage and checking the DB names, which already include the prefix.

Reply:
Ah sorry, to clarify I figured we could just test if the 'Untitled' database exists (the only one I expect anyone has currently), and if so call something like `createProject('Untitled', 'Untitled')` to get it into the `userProjects` store - as-is this wouldn't work, since the IDBStoredProject would then try to access `SplashKitOnlineProject_Untitled`. 
Migrating old projects isn't a priority though (I don't think anyone is really using this yet), so just the constant is fine 😄

Reply:
Ah, I understand what you mean; incorporating it would avoid the code looking for a DB named 'SplashKitOnlineUserProject_Untitled' if we wanted to fill-in the last opened project as 'Untitled' in order to migrate easily. We could do that, but I chose to add the prefix specifically so that accessing a project DB could never accidentally access some other DB in storage; it's a kind of sanitisation, I suppose.